### PR TITLE
#49 feat: created medium casts and 15 random media endpoint

### PR DESCRIPTION
**Pull request by fkula5**

### CODE REVIEW

File: app/Http/Controllers/MediumController.php [hunk 1]
```diff
@@ -28,7 +31,8 @@ public function __construct(MediumService $mediumService)
      */
     public function index()
     {
-        //
+        $media = Medium::all()->random(15);
+        return $this->sendResponse(new MediumCollection($media), 'Medium retrieved successfully.');
```
Comment (marioooo0o) [inline app/Http/Controllers/MediumController.php:58]: Should be other message

File: app/Http/Resources/CastResource.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    public function toArray($request)
+    {
+        //return parent::toArray($request);
+        return [
+            'id' => $this->id,
+            'medium_id' => $this->medium_id,
+            'person_id' => $this->person_id,
+            'gender_id' => $this->gender_id,
+            'character' => $this->character,
+            'created_at' => $this->created_at,
```
Comment (marioooo0o) [inline app/Http/Resources/CastResource.php]: change format returned date

File: app/Http/Resources/CastResource.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            'updated_at' => $this->updated_at,
```
Comment (marioooo0o) [inline app/Http/Resources/CastResource.php]: same

File: app/Http/Resources/CastResource.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        //return parent::toArray($request);
```
Comment (marioooo0o) [inline app/Http/Resources/CastResource.php]: clean

**Review by marioooo0o — CHANGES_REQUESTED** [review]
please fix these things and add swagger to this endpoints

File: app/Http/Resources/CastResource.php (hunk 4 above)
Comment (fkula5) [reply app/Http/Resources/CastResource.php]: You mean that everything is fine or remove commented line ?

Comment (marioooo0o) [reply app/Http/Resources/CastResource.php]: Remove commented line

File: app/Http/Resources/CastResource.php (hunk 2 above)
Comment (fkula5) [reply app/Http/Resources/CastResource.php]: how should the date look ?

Comment (fkula5) [reply app/Http/Resources/CastResource.php]: and whether do it in model or just in resource?

Comment (marioooo0o) [reply app/Http/Resources/CastResource.php]: Maybe only date without time part
Please add this in resource

Comment (marioooo0o) [reply app/Http/Resources/CastResource.php]: See if there is a date anywhere on the mock-ups and adjust the format to it if you think it's ok

**Pull request merged**